src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1145,29 +1145,23 @@ namespace WaterToAirHeatPumpSimple {
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).WaterMassFlowRate =
                 state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).DesignWaterMassFlowRate;
```

Comment:
@rraustad This line makes me concerned whether water mass flow rate was set correctly. My test file is running at rated water flow rate, I have no idea how it will behave in a test file where water is not at rated. I assume it is possible that pump flow rate can be off-rated, right? so far by the observation I saw it can mess up the coil water side reporting.

Reply:
This coil model does run at the design water flow rate. The parent knows nothing of the water-side flow. Only plant limitations would affect the water flow rate. There is a switch to either run a full water flow or cycle the flow based on PLR. It would be good to check this during testing where there is an energy balance between air- and water-side.

    if ((state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).WaterCyclingMode) == WaterCycling) {
        // plant can lock flow at coil water inlet node, use design flow multiplied by PLR to calculate water mass flow rate
        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).WaterMassFlowRate =
                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).DesignWaterMassFlowRate * PartLoadRatio;


WaterCyclingMode is set by the parent via SetSimpleWSHPData.

    ZoneHVAC:WaterToAirHeatPump,
     A19, \field Heat Pump Coil Water Flow Mode
        \type choice
        \key Constant
        \key Cycling
        \key ConstantOnDemand
        \default Cycling
        \note used only when the heat pump coils are of the type WaterToAirHeatPump:EquationFit
        \note Constant results in 100% water flow regardless of compressor PLR
        \note Cycling results in water flow that matches compressor PLR
        \note ConstantOnDemand results in 100% water flow whenever the coil is on, but is 0% whenever the coil has no load